src/com/esotericsoftware/kryo/Kryo.java

```diff
@@ -149,7 +83,7 @@
 	private Object needsCopyReference;
 	private GenericsResolver genericsResolver = new GenericsResolver();
 
-	private FieldSerializerConfig fieldSerializerConfig = new FieldSerializerConfig();
+	private TaggedFieldSerializerConfig fieldSerializerConfig = new TaggedFieldSerializerConfig();

```

Comment:
Actually there should be both `FieldSerializerConfig fieldSerializerConfig` and `TaggedFieldSerializerConfig taggedFieldSerializerConfig`, because this are 2 different serializers and 2 different configs.